src/components/AssetsTable/AssetsTable.test.jsx

```diff
@@ -81,9 +81,21 @@ const numberOfImages = defaultProps.assetsList.filter(asset => asset.thumbnail).
 const numberOfStudioButtons = defaultProps.assetsList.filter(asset => asset.portable_url).length;
 const numberOfWebButtons = defaultProps.assetsList.filter(asset => asset.external_url).length;
 
+let parentElement;
+let querySelector;
 let wrapper;
 
 describe('<AssetsTable />', () => {
+  beforeEach(() => {
```

Comment:
It would be great if you would either move the contents of the beforeEach and afterEach into functions with descriptive names or leave a comment explaining what this is for. I can see it being confusing in a few months from now with fresh eyes.